lib/api/firebase.ts

```diff
@@ -31,15 +31,11 @@ export type App = admin.app.App;
  * @see {@link https://stackoverflow.com/a/41044630/10023158}
  * @see {@link https://stackoverflow.com/a/50376092/10023158}
  */
+
 const firebase = admin.initializeApp(
   {
-    credential: admin.credential.cert({
-      projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
-      privateKey: (process.env.FIREBASE_ADMIN_KEY || '').replace(/\\n/g, '\n'),
-      clientEmail: process.env.FIREBASE_ADMIN_CLIENT_EMAIL,
-    }),
+    credential: admin.credential.applicationDefault(),
```

Comment:
Don't add these changes, remember? You should add them to a `.gitignore` file or something so you don't accidentally commit them.